src/enlyze/client.py

```diff
@@ -233,10 +228,10 @@ def _get_timeseries(
                 machine_uuid=machine_uuid,
                 start=start,
                 end=end,
-                variables=chunk,
+                variables=dict(variable_chunk),
                 resampling_interval=resampling_interval,
             )
-            for chunk in chunks
+            for variable_chunk in chunks
```

Comment:
either for `chunk in chunk` or `variable_chunk in variable_chunks` 😄

Reply:
7d83d7f